idea-plugin/build.gradle.kts

```diff
@@ -33,7 +33,13 @@ dependencies {
 }
 
 intellijPlatform {
-    pluginConfiguration { name = "Compose Multiplatform IDE Support" }
+    pluginConfiguration {
+        name = "Compose Multiplatform IDE Support"
+        ideaVersion {
+            sinceBuild = "243.21565"
```

Comment:
It won't be compatible with Android Studio. The last stable in Toolbox seems `Build #AI-242.23339.11.2421.12700392`.

Can we use it?

Reply:
Thanks, nice catch!
The latest stable Android Studio doesn't enable K2 mode by default, so it's less of an issue there.